### PR TITLE
Fixing hybrid android apps (and moving to cordova android 10.1.1)

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: tools/postinstall-android.js [hunk 1]
```diff
@@ -96,24 +96,10 @@ if (data.indexOf("SalesforceHybrid") < 0)
     var oldAndroidDepTree = "android {";
     var newAndroidDepTree = "android {\n\tpackagingOptions {\n\t\texclude 'META-INF/LICENSE'\n\t\texclude 'META-INF/LICENSE.txt'\n\t\texclude 'META-INF/DEPENDENCIES'\n\t\texclude 'META-INF/NOTICE'\n\t}";
     replaceTextInFile(path.join(appProjectRoot, 'app', 'build.gradle'), oldAndroidDepTree, newAndroidDepTree);
-    var oldGradleToolsVersion = "com.android.tools.build:gradle:4.0.0";
```
Comment (wmathurin) [inline tools/postinstall-android.js:99]: App generated by cordova uses 7.

File: tools/postinstall-android.js [hunk 2]
```diff
@@ -96,24 +96,10 @@ if (data.indexOf("SalesforceHybrid") < 0)
...
 }
 
-// Replacing values in top level build.gradle to avoid conflicts in Gradle builds.
-console.log('Fixing project workspace build.gradle');
-var oldGradleToolsVersion = "com.android.tools.build:gradle:4.0.0";
-var newGradleToolsVersion = "com.android.tools.build:gradle:7.0.4";
-
-replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), oldGradleToolsVersion, newGradleToolsVersion);
-replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), /defaultBuildToolsVersion=.*/, '');
-replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), /defaultMinSdkVersion=.*/, 'defaultMinSdkVersion=24');
```
Comment (wmathurin) [inline tools/postinstall-android.js:113]: min and target sdk are specified through plugin.xml

**Comment by wmathurin** [comment]
Merging, I need this merged for tests to run successfully on https://github.com/forcedotcom/SalesforceMobileSDK-Package/pull/249.

File: tools/postinstall-android.js (hunk 2 above)
Comment (bhariharan) [reply tools/postinstall-android.js:113]: Nice to know that we  can finally get rid of more post-install steps.

**Pull request merged**